pkg/common/utils_test.go

```diff
@@ -588,73 +588,91 @@ func TestConvertGiBStringToInt64(t *testing.T) {
 		{
 			"valid number string",
 			"10000",
-			1,
+			10000,
 			false,
 		},
 		{
-			"round Ki to GiB",
-			"1000Ki",
-			1,
+			"round M to number",
+			"1M",
+			1000000,
 			false,
 		},
 		{
-			"round k to GiB",
-			"1000k",
+			"round m to number",
+			"1m",
 			1,
 			false,
 		},
 		{
-			"round Mi to GiB",
-			"1000Mi",
-			1,
+			"round k to number",
+			"1k",
+			1000,
 			false,
 		},
 		{
-			"round M to GiB",
-			"1000M",
-			1,
-			false,
+			"invalid empty string",
+			"",
+			10000,
+			true,
 		},
 		{
-			"round G to GiB",
-			"1000G",
-			932,
+			"invalid string",
```

Comment:
do you think it's worth it to add a case for `1KiB` or `1GB` as invalid string?

Reply:
We can add that.